projects/module-3/rancid-tomatillos-v2.md

```diff
@@ -0,0 +1,240 @@
+* Test Redux functions
+
+## Iterations
+
+### Iteration 0 - Day-1 Deliverables
+
+Create a group Slack channel and include your instructors in the channel:
+  * Layouts/Wireframes sketches of your user interface
+  * Group DTR
+  * Project management tool (GitHub Projects, Trello, etc.) - be sure this is public so your instructors can view it
+  * PR template your group agrees to follow for every PR submitted ([here are some instructions from GitHub on how to create this](https://help.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository))
+
+After you send us this information, we'll give you your user's name, email, and password.
+
+### Iteration 1 - User Info, API Intro, and Movies
+
+You will be assigned a user for your pair. Each user has a name, email, and password that you will use throughout the project. This is your team's user!
+
+At this point, get your head around the API's endpoints via the documentation below. Try them out in Postman - you don't need an API key for this! See what you can get from all the endpoints available.
+
+For your app, create a homepage where the visitor can see all of the movies in the database with their average ratings. When on this page, the url's path will be `/` (ex: in development, the url will look something like: `http://localhost:3000/`).
+
+### Iteration 2 - User Login
+
+Create a separate login page where your user can login. When a user navigates to this page, the URL's path should be `/login` (ex: in development, the url will be `http://localhost:3000/login`).
+
+Ultimately, you'll need to obtain the user's unique ID value to make requests later. Logging in the user using the POST `/login` route documented below will give you that ID value.
+
+It probably seems weird to POST to `/login` to login a user, and this endpoint isn't even creating anything! Using a POST verb for logging in is a common convention because we need to send sensitive information like passwords in the body of the login request, and a POST lets us do that. Again, nothing is created in the DB for logging in - you are doing this to retrieve user information (the ID) to use while the user is logged in.
+
+After logging in, the user knows they are logged in and they are taken back to the homepage.
+
+A user should be able to logout, and they are taken to the homepage. The app should show that no one is logged in.
+
+At this point, you should have your testing suite up and going. Get tests running, and test your Login workflow as it stands now. Yes, your tests might change, but that's ok and expected.
+
+
+### Iteration 3 - Movie Show Page
+
+Create a dynamic route for a "show page" for each movie (the show page for a movie with an id of 5 would be at a url path of `/movies/5`, for instance). A show page is a app page that shows only one particular item. In this case, the show page is for a particular movie. This page should show all information available about the movie.
```

Comment:
Router is going to be needed for this bit of functionality, right?

Reply:
I guess I'm not sure how to edit React urls without Router ...

Reply:
Yep, if we want them to make dynamic urls (without going down the weird rabbit holes you mentioned), they'll need to know about Router to do this.

Reply:
You may be able to do it with regular <a> tags, but it probably requires some more webpack configuration now that I think about it...